packages/docs/docs/self-hosting/install-on-ubuntu.md

```diff
@@ -126,12 +126,167 @@ Install dependencies
 npm ci
 ```
 
-Build the server and necessary dependencies
+Build the server, app, and necessary dependencies
+
+```bash
+npm run build:fast
+```
+
+## Start Medplum
+
+:::info
+
+These are abbreviated instructions. For full details, see [Run the stack](/docs/contributing/run-the-stack)
+
+:::
+
+In one terminal, start the `server` in development mode:
+
+```bash
+cd packages/server
+npm run dev
+```
+
+In another terminal, start the `app` in development mode:
+
+```bash
+cd packages/app
+npm run dev
+```
+
+You should now be able to access the Medplum app at [http://localhost:3000](http://localhost:3000).
+
+## Optional: Add nginx as a reverse proxy
+
+To run Medplum securely, you should use SSL/TLS via reverse proxy such as nginx.
+
+Install nginx and Certbot:
+
+```bash
+sudo apt-get install nginx certbot python3-certbot-nginx
+```
+
+Before setting up SSL, make sure your domains point to your server and Nginx is running:
+
+```bash
+sudo systemctl start nginx
+sudo systemctl enable nginx
+```
+
+Get SSL certificates from Let's Encrypt for both domains:
+
+```bash
+sudo certbot --nginx -d app.example.com
+sudo certbot --nginx -d api.example.com
+```
+
+Create an `app` config file such as `/etc/nginx/sites-available/app.example.com`:
+
+```nginx
+# /etc/nginx/sites-available/app.example.com
+server {
+    listen 80;
+    listen [::]:80;
+    server_name app.example.com;
+
+    # Redirect HTTP to HTTPS
+    location / {
+        return 301 https://$server_name$request_uri;
+    }
+}
+
+server {
+    listen 443 ssl;
+    listen [::]:443 ssl;
+    server_name app.example.com;
+
+    ssl_certificate /etc/letsencrypt/live/app.example.com/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/app.example.com/privkey.pem;
+
+    location / {
+        proxy_pass http://localhost:3000;
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection 'upgrade';
+        proxy_set_header Host $host;
+        proxy_cache_bypass $http_upgrade;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+    }
+}
+```
+
+Create a `api` config file such as `/etc/nginx/sites-available/api.example.com`:
+
+```nginx
+# /etc/nginx/sites-available/api.example.com
+server {
+    listen 80;
+    listen [::]:80;
+    server_name api.example.com;
+
+    # Redirect HTTP to HTTPS
+    location / {
+        return 301 https://$server_name$request_uri;
+    }
+}
+
+server {
```

Comment:
Did we make sure this works for WebSockets? When I get back to my laptop I can check but I remember last time I set up nginx with WebSockets I had to do something special

Reply:
Actually I think this is fine, I think I might be thinking about tuning buffers which is a separate issue. LGTM